### ISSUE TITLE
Figure out how to get official course data

### ISSUE THREAD

**Issue by khanny17** [rit-sse/CoursePlanner#53]
It would be wonderful if we could get official course or plan data for specific schools (like RIT) so we could let users easily search to add courses or even search for official plans rather than publicly shared plans. Precisely how we should go about doing that remains an open question.

**Comment by BenAlderfer** [comment]
We could talk to CSH about getting RIT's data. I could talk to a member about getting connected with them if you want

**Comment by khanny17** [comment]
Yeah sure ask them how they do it, where they store it, and if we could collaborate

**Comment by MoralCode** [comment]
https://api.rit.edu
